tabwriter/tabwriter_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"bytes"
 	"testing"
 
-	"github.com/influxdata/influx-cli/v2/internal/tabwriter"
+	"github.com/influxdata/influx-cli/v2/tabwriter"
```

Comment:
Can we stick it under `pkg/`? That's where all the other random utils currently live

Reply:
Good point. Done